src/client/PyCallbacks.h

```diff
@@ -157,16 +171,31 @@ class PyCallbacks : public IPyCallbacks {
 		return prt::STATUS_OK;
 	}
 
-	prt::Status attrBool(size_t /*isIndex*/, int32_t /*shapeID*/, const wchar_t* /*key*/, bool /*value*/) override {
+	prt::Status attrBool(size_t isIndex, int32_t /*shapeID*/, const wchar_t* key, bool value) override {
+		if (mRuleFileInfo && !isHiddenAttribute(mRuleFileInfo, key)) {

```

Comment:
please deduplicate this block of code (maybe using a template), it seems to be the same for all three attribute data types